### ISSUE TITLE
Importing Load Effects from a FEM software to IDEA [QUESTION]

### ISSUE THREAD

**Issue by Fardadmokhtari** [idea-statica/iom-examples#81]
Hello,

I am trying to import load effects and results of an FEM analysis to IDEA StatiCa. I don't want to define all the load cases and combinations again in IDEA. I only want to import the load combination results without defining the load cases and the combination factors because the FEM software has done all that on its end.

I found that in the "LoadingType" enumeration we have 3 different types: 1. LoadCase 2. Combination 3. ResultClass
Is the ResultClass created for this intention?

If yes, the properties of this class were a bit confusing to me. Do you have an example explaining these properties and their implementation?

My second question: My load results are defined based on element1ds in the FEM software and Member1D is defined in IDEA IOM however my FEM software doesn't understand the Member1Ds to modify the results based on that. My question is that can I define the loads based on element1Ds?

Is the following modification (the Bold part) in the code enough for this purpose or is there something else that I should be aware of?
var resultOnMember = new ResultOnMember
            {
                ResultType = ResultType.InternalForces,
                Member = new IdeaRS.OpenModel.Result.Member(**_MemberType.Element1D_**, ID),
                Results = CreateResultBases(memberID, sFrameData.LoadStationList, sFrameData)
            };

By the way, I don't know the following information is important to you or not but, my FEM software is S-Frame and the results are saved in an access database file. I am reading the access database file and try to automate importing the results into IDEA. This is the format that my results are saved in the access database:
![image](https://user-images.githubusercontent.com/55615235/114514723-d3a60680-9bf8-11eb-85f0-220894b5fe6f.png)

Thanks for your time and consideration,

Regards,
Fardad

**Comment by Fardadmokhtari** [comment]
Hello,

Have you had the chance to look into my issue?

Thank you for your time and consideration,
Regards,
Fardad

**Comment by mpospisil** [comment]
Hi Fardad,

Not yet. I'm sorry. It is still in our backlog. 

Martin

**Comment by Fardadmokhtari** [comment]
Hi Martin,

Please let me know if you think my question is too long, and I should submit a summary of it. I would appreciate your time.

Thank you,
Fardad

**Comment by Fardadmokhtari** [comment]
Hello,

Have you had the chance to look into my issue?

Thank you for your time and consideration,
Regards,
Fardad

**Comment by tLuber72** [comment]
Hello,
Our results are based on definition of Loadcase and than we need to define Combination with factors.
But you can insert into results of loadcase you results of Combination. In that case it is not neccesary to define Combinations

ResultClass is envelop of Load case or Combination. It to not neccessary to define on your side.

"My second question: My load results are defined based on element1ds in the FEM software and Member1D is defined in IDEA IOM however my FEM software doesn't understand the Member1Ds to modify the results based on that. My question is that can I define the loads based on element1Ds?"

If you have only Element1D you have to create Member1D too. With the same Id.  One to One. Member1D will contains one Element1D

"Is the following modification (the Bold part) in the code enough for this purpose or is there something else that I should be aware of?
var resultOnMember = new ResultOnMember
{
ResultType = ResultType.InternalForces,
Member = new IdeaRS.OpenModel.Result.Member(MemberType.Element1D, ID),
Results = CreateResultBases(memberID, sFrameData.LoadStationList, sFrameData)
};"
It Is OK.

Regards Tom

**Comment by Fardadmokhtari** [comment]
Hello Tom,

Thank you for your response. I got your point for my first question, but still, I have a question about the second question.

I have defined the member1Ds in the open model. The problem is that I want to define the loads based on element1Ds because the FEM software exports the loading information based on the original elements, which are equivalent to element1Ds in IDEA StatiCa. The FEM solver doesn't understand member1Ds in order to convert and export the results according to member1D. Is there a way to define the loads on element1Ds instead of member1Ds? I have created member1Ds in OpenModel, but my question is about the OpenModelResult object. What is the easiest way to deal with this problem?

Thank you,
Fardad
